lib/flight_job/commands/list_job_results.rb

```diff
@@ -37,7 +37,13 @@ class ListJobResults < Command
 
       def run
         job = load_job(args.first)
-        assert_results_dir_exists(job)
+
+        # Skip the assertion check when hard wrapping and the directory exists
+        # This causes to command to run on empty directories
+        if hard_wrap? && Dir.exists?(job.results_dir)
```

Comment:
I don't think this is doing what you want it to.

Reply:
Also `hard_wrap` is a confusing name.  I'm not sure what part of passing additional options to the `ls` command is hard or why its wrapped.  Or is hard wrap the opposite of a soft wrap?

Can we get a better name for this please?

Also let's rename `HARD_ERROR` to `LS_ERROR`.

Reply:
Also, why do we want this branch.  `assert_results_dir_exists` doesn't check for empty directories.

Reply:
I've added the assertion check back here: 57a3f08de3ee67b0da9e0992f505c22fad679b9b.

I originally removed the check when hard wrapping `ls` to preserve the original error. However this isn't required.

Reply:
Renamed `LS_ERROR`: 583e9de2742b3684a28248ece69a625c1ab8674f